packages/web-components/fast-element/tsconfig.json

```diff
@@ -1,16 +1,20 @@
 {
     "extends": "../../../tsconfig.json",
     "compilerOptions": {
-        "outDir": "dist",
-        "baseUrl": ".",
+        "declarationDir": "dist/dts",
+        "outDir": "dist/esm",
         "strictNullChecks": true,
-        "target": "es2015"
+        "target": "es2015",
+        "module": "ESNext",
+        "removeComments": true,
```

Comment:
Isn't this is going to remove jsDoc comments documenting the feature/API that is surfaced in IDEs?

Reply:
Good point. How should we handle this? Would it be ok to have the comments in dts only but not in the distributed source? If so, we can use this plugin: https://github.com/Swatinem/rollup-plugin-dts to build a single dts file for the library. Or should we leave the comments in both the esm and dts builds but just remove them from the single script-file build?

Reply:
If the goal was just to remove them from the script file I think the latter is the most straight-forward. I'd rather leave it to consuming apps to run their bundles through a minifier if they don't want comments.

Reply:
Ok fixed.